src/controllers/connection/__tests__/index.test.ts

```diff
@@ -4,10 +4,14 @@ import sinon from 'sinon'
 
 import { toHTMLString, withConnectionMocks } from './helpers.js'
 
+import { Request } from 'express'
+import { mockLogger } from '../../__tests__/helpers.js'
 import { ConnectionController } from '../index.js'
 import { validConnection } from './fixtures.js'
 
 describe('ConnectionController', () => {
+  const req: object = { log: mockLogger }
```

Comment:
If you define this as a `Request` and do the `as Request` here you'll remove duplication in the rest of the changeset

Reply:
nice one!